_posts/en/newsletters/2025-11-07-newsletter.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Bitcoin Optech Newsletter #379"
+permalink: /en/newsletters/2025/11/07/
+name: 2025-11-07-newsletter
+slug: 2025-11-07-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+
+FIXME:bitschmidty
+
+## News
+
+FIXME:bitschmidty
+
+- **Comparing performance of ECDSA signature validation in OpenSSL vs. libsecp256k1** Sebastian
+  Falbesoner [posted to Delving][sebastion delving] about comparing the performance of ECDSA
+  signature validation between OpenSSL and libsecp256k1 over the last decade. He mentions that it
+  will be the 10th anniversary of Bitcoin Core switching from OpenSSL to libsecp256k1, and wanted to
+  imagine a hypothetical situation in which Bitcoin Core never switched.
+
+  From the start, when the switch happened, libsecp256k1 was 2.5 - 5.5 times faster than OpenSSL.
+  Nonetheless, OpenSSL could have improved over the years, so it was worth testing to see how it
+  compared over the decade.
+
+  The methodology was tested using 3 states that can be tested with functions in both libraries. A
+  benchmark was made with a list of pseudo-random key pairs. He ran the benchmark on 3 separate
+  machines and provided a bar plot. The chart showed that over the years, libsecp256k1 had improved
+  significantly while OpenSSL had remained the same.
```

Comment:
Maybe provide a tangible example of significantly here @kevkevinpal

Reply:
thanks pushed fix in [df88b3b](https://github.com/bitcoinops/bitcoinops.github.io/pull/2544/commits/df88b3b9ff66287bd5bcd21de2ecf608013e7056)